web/themes/joinup/templates/block/block--separated.html.twig

```diff
@@ -0,0 +1,45 @@
+{#
```

Comment:
The markup of this template is the same as the original blocks, are you sure a preprocess can't attach the necessary classes without a template?